scene/gui/color_picker.cpp

```diff
@@ -35,9 +35,14 @@
 #include "core/math/color.h"
 #include "core/os/keyboard.h"
 #include "core/os/os.h"
+#ifdef TOOLS_ENABLED
+#include "editor/editor_quick_open.h"
```

Comment:
You can't include editor code in scene/ files. You'll need to find a way to use quick open without the dependency (maybe it can be handled by parent node and a callback).